src/zabapgit_authorizations.prog.abap

```diff
@@ -0,0 +1,49 @@
+*&---------------------------------------------------------------------*
+*&  Include           ZABAPGIT_AUTHORIZATIONS
+*&---------------------------------------------------------------------*
+
+TYPES: ty_activity TYPE string.
+
+CONSTANTS: BEGIN OF gc_activities,
+             uninstall TYPE ty_activity VALUE 'UNINSTALL',
+           END OF gc_activities.
```

Comment:
imho `activities` is not the best name here. I'd call `ty_auth, gc_auth` or maybe `ty_acl` (access control list) or maybe `auth_rule`. But `activity` can mean anything.